legoshop.py

```diff
@@ -537,11 +576,13 @@ def process(self, lego_set, order_list):
         self._init_browser(self.browser_name,
                            self.lego_shop + "/service/replacementparts/sale?chosenFlow=3")
         self._process_survey()
+        self._process_age_gate()
         self._process_cookies_accept()
-        self._process_survey_age_country()
+        self._process_choose_country()
 
-        if not self._process_login():
-            return
+        # The site now fails these login attempts as they are too fast.
+        # if not self._process_login():
+        #     return
```

Comment:
Hmmm. Should we remove this code? Or can we do something about it? (Print some log output that explains what happens?)